packages/common/src/stream.ts

```diff
@@ -120,6 +123,7 @@ export interface LogEntry {
  * to load anything else from ipfs or the p2p Ceramic network.
  */
 export interface CommitData extends LogEntry {
+  // TODO use latest naming convention genesis -> init, signed -> data, anchor -> time
```

Comment:
I think this comment can be removed, nothing here is actually using the old naming convention

Reply:
I added it because some properties inside that interface use the previous convention